.github/workflows/mlmodeldownloader.yml

```diff
@@ -65,9 +64,7 @@ jobs:
 
   catalyst:
     # Don't run on private repo unless it is a PR.
-    # TODO: manjanac@ Uncomment line below to re-enable catalyst.
-    # if: github.repository != 'FirebasePrivate/firebase-ios-sdk' || github.event_name == 'pull_request'
-    if: false
+    if: github.repository != 'FirebasePrivate/firebase-ios-sdk' || github.event_name == 'pull_request'
     runs-on: macOS-latest
     steps:
     - uses: actions/checkout@v2
```

Comment:
Change the `build` on line 74 to `test` to also run tests

Reply:
I made this change, but my integration tests are failing. Am I missing the GoogleService-Info.plist under `catalyst`?

Reply:
Usually a `GoogleService-Info.plist` isn't needed for unit tests. It looks like the scheme needs to be specified - like https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/core.yml#L70

Reply:
Ah perfect, thanks!